examples/plot_nonstatconvolve.py

```diff
@@ -116,16 +115,18 @@
 )
 
 x = np.zeros((nx, nz))
-x[:, 201] = 1.0
-x[:, 401] = -1.0
+line1 = (np.arange(nx) * np.tan(np.deg2rad(25))).astype(int) + (nz - 1) // 4
+line2 = (np.arange(nx) * np.tan(np.deg2rad(-25))).astype(int) + (3 * (nz - 1)) // 4
+x[np.arange(nx), np.clip(line1, 0, nz - 1)] = 1.0
+x[np.arange(nx), np.clip(line2, 0, nz - 1)] = -1.0
```

Comment:
This makes the lines at an angle. It allows a better visualization of the nonstationarity of the events

Reply:
Good idea